vue/src/components/SettingsPanel.vue

```diff
@@ -5,11 +5,11 @@ import type { Ref } from "vue";
 import { state } from "../store";
 import { updatePattern } from "../interactions/fillPatterns";
 
-const showSiteOutline: Ref<boolean> = ref(false);
-const groundTruthPattern: Ref<boolean> = ref(false);
-const otherPattern: Ref<boolean> = ref(false);
-const patternThickness = ref(1);
-const patternOpacity = ref(255);
+const showSiteOutline: Ref<boolean> = ref(state.filters.showSiteOutline || false);
+const groundTruthPattern: Ref<boolean> = ref(state.filters.groundTruthPattern || false);
+const otherPattern: Ref<boolean> = ref(state.filters.otherPattern || false);
+const patternThickness = ref(state.patterns?.patternThickness || 1);
+const patternOpacity = ref(state.patterns?.patternOpacity || 255);
```

Comment:
Instead of allocating new refs, and having watchers updating the "real" values in the `state` object, why not make these `computed` and have a computed setter for modifying the `state.*` values directly?

Reply:
Good point, I was just patching the issue but this makes more sense to do.